app/config/translations.json

```diff
@@ -126,13 +141,19 @@
     "Fin Source": "Funds"
   },
   "es": {
```

Comment:
I'm not sure. Let's discuss it. We currently have two sources of translation strings: Google spreadsheet (for `en` and `es`), and local `.json` files for all other languages. Suppose we'll change that python script to export data from spreadsheet to local files, and then use that local files. It's bad idea - instead of one auto-generated file we will have two and they will be located in the same folder as manually added ones. Another solution is to run python script as a part for build process; it's good for production, but not so great for development when `npm run build` may be running frequently. If you have other ideas - please put them in a comment.